lib/manager/npm/post-update/yarn.spec.ts

```diff
@@ -94,6 +94,28 @@ describe(getName(__filename), () => {
       expect(fixSnapshots(execSnapshots)).toMatchSnapshot();
     }
   );
+  it.each([['1.22.0']])(
```

Comment:
please do not use `it.each` for a single value. It has no benefit and will slow down tests.

I know it's from copy'n'paste. 🙃

Reply:
After removing the null check, there was nothing this test covered on top of the updated snapshot tests, so I removed it: https://github.com/renovatebot/renovate/pull/6759/commits/f31a7e7e35b005fda70fc2be1faae70c5632e8c0 - let me know if there's an additional test that should be included though.